src/main/java/com/spotify/docker/Utils.java

```diff
@@ -64,5 +66,19 @@ public static void pushImage(DockerClient docker, String imageName, Log log)
       log.info("Pushing " + imageName);
       docker.push(imageName, new AnsiProgressHandler());
   }
+  
+  public static void removeImage(DockerClient docker, String imageName, Log log)
```

Comment:
Maybe add a javadoc or comment here to say that `removeImage(_, true, false)` will force image removal and not delete untagged parents?